test/geometry/test_stereo.py

```diff
@@ -0,0 +1,198 @@
+from typing import Type
+
+import pytest
+import torch
+from torch.testing import assert_allclose
+
+import kornia
```

Comment:
we try to import the full paths in testing to ensure everything can be correctly imported

Reply:
Got it!